### PR TITLE
Add basic layout and navigation to member page

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add basic, responsive layout and navigation to member page 

Based of first layout sketched https://excalidraw.com/#json=ItdKOGnTB88HbDOqObzz_,vqcGBFqoOC3yVPCOvzkSgA

https://github.com/HedvigInsurance/racoon/assets/6661511/dbf8d3e2-b30e-4dac-a5b3-7a80df305d5d

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3331** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3331" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3331?utm_source=stack-comment).

File: apps/store/src/features/memberArea/MemberPage.tsx [hunk 1]
```diff
@@ -16,19 +22,37 @@ export const MemberPage = () => {
...
+        <Navigation>
+          <NavigationList>
+            <li>
+              <Link href={'/member'}>Insurances</Link>
+            </li>
+            <li>
+              <Link href={'/member/profile'}>Profile</Link>
+            </li>
+            <li>
+              <Link href={'/member/claim'}>Make a claim</Link>
```
Comment (alebedev) [inline apps/store/src/features/memberArea/MemberPage.tsx:42]: Should we disable things that don't work yet?  And this would be app deep link, right?

**Review by alebedev — APPROVED** [review]
Nice!

File: apps/store/src/features/memberArea/MemberPage.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/features/memberArea/MemberPage.tsx:42]: Yeah let's remove the link for now and just have a placeholder 👍 @alebedev

**Comment by gustaveen** [comment]
### Merge Activity

* **Oct 4, 7:58 AM**: @gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3331).

**Pull request merged**